_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -3433,7 +3433,8 @@
 	movement_type = 1;
 	name = "Sergeant Araneus";
 	real_name = "Sergeant Araneus";
-	response_help = "pets";
+	response_help_continuous = "pets";
+	response_help_simple = "pet";
```

Comment:
Can I change Sergeant Araneus's variables in MetaStation.dmm like so?

Reply:
This is fine. Should get own path really but that's not in this pr scope.